httpx/concurrency.py

```diff
@@ -244,31 +247,74 @@ def run(
         finally:
             self._loop = loop
 
+    def create_event(self) -> BaseEvent:
+        return asyncio.Event()  # type: ignore
```

Comment:
I saw a nice way to do this with `typing.cast(...)` so we don't have to skip the type checking.

Reply:
Good point! I always forget about `.cast()`.